modules/modal/react/stories/stories.tsx

```diff
@@ -1,13 +1,43 @@
 /// <reference path="../../../../typings.d.ts" />
 import * as React from 'react';
-import {storiesOf} from '@storybook/react';
+
 import withReadme from 'storybook-readme/with-readme';
 
+import {Modal, useModal} from '@workday/canvas-kit-react-modal';
+
 import {Button, DeleteButton} from '../../../button/react';
-import Modal, {useModal} from '..';
+import {FormField} from '../../../form-field/react';
+import {TextInput} from '../../../text-input/react';
```

Comment:
There is examples of both relative imports and package name imports in stories. Personally I think we should do only package name imports, but it is annoying to maintain `devDependencies` that are only added for stories. I think we should have a lint rule for only package name imports in stories and remove the requirement for `devDependencies` to reflect what's in story files.